openstack/keystone/values.yaml

```diff
@@ -464,3 +464,14 @@ sapcc_rate_limit:
   enabled: false
   persistence:
     enabled: false
+
+2fa:
+  enabled: false
+  replicaCount: 3
```

Comment:
Why are there 3 replicas? Does it make sense to make the number of replicas equal to the number of keystone-api pods?

Reply:
We can make it match, but I think the throughput, memory/cpu and overall performance behaviour of this proxy is completely different then keystone. it only parses json and make a local decision to forward the request. I think the main motivation of multiple replicase is fault tolerance for this proxy and to allow for a smooth rolling update of a new version.
If make it match then I think it would make sense to run this as sidecar of the api pod to also avoid network roundtrips inside of the cluster.